bitrise.yml

```diff
@@ -232,6 +232,12 @@ workflows:
     - deploy-to-bitrise-io@2:
         inputs:
         - notify_email_list: $NOTIFY_EMAILS
+    - slack@3:
+        is_always_run: false
+        inputs:
+        - channel: '#tm-mobile-builds'
+        - text: iOS Build Succeeded!
```

Comment:
Will some information about the build be added as well?

Reply:
You can the similar message [there](https://salemove.slack.com/archives/C01E6TP4VB8/p1657193924142039)